### ISSUE TITLE
Add Windows to Templates?

### ISSUE THREAD

**Issue by yachub** [genebean/packer-templates#4]
Would it be possible to add Windows boxes/containers to the template builds, and if so, would you consider adding that functionality?

Personally, I would be interested in using a "windows-2016-puppet5" vagrant box that includes Chocolatey and the Puppet Agent for testing/building Puppet code. Sounds like a cool thing to me anyways.

I would be happy to help as well if needed.

Thanks,

**Comment by genebean** [comment]
Certainly. I have been playing around with [StefanScherer/packer-windows](https://github.com/StefanScherer/packer-windows) and think I could adapt what he has for this. Naturally, PR's are welcome if you beat me to it :)
